proposals/ard_normalized_radar_backscatter.json

```diff
@@ -13,8 +13,20 @@
             "name": "data",
             "description": "The source data cube containing SAR input.",
             "schema": {
-                "subtype": "raster-cube",
-                "type": "object"
+                "type": "object",
+                "subtype": "datacube",
+                "dimensions": [
```

Comment:
Which dimensions are required? Is temporal required?

-> Alex + Michele tend towards no